### PR TITLE
upload and run in container works

### PR DESCRIPTION
**Pull request by jchorl**

### CODE REVIEW

**Comment by rwongone** [comment]
LGTM

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,6 @@ clean:
 	-docker rm -f collabtest 2> /dev/null || true
 
 run:
-	docker run -it --rm -p 8080:8080 -v /var/run/docker.sock:/var/run/docker.sock --name collabtest collabtest/collabtest
```
Comment (rwongone) [inline Makefile]: entrypoint?

File: handlers/handlers.go [hunk 2]
```diff
@@ -1,69 +1,160 @@
...
+		logrus.WithError(err).Error("Could not create docker client")
+		return err
+	}
+
+	// cannot take address of const int
+	timeout := BUILD_TIMEOUT
+
+	// TODO actually select correct image
+	containerConfig := &container.Config{
+		Image:           "gcc",
```
Comment (rwongone) [inline handlers/handlers.go:60]: Try frolvlad/alpine-gcc?

Comment (jchorl) [reply handlers/handlers.go:60]: Yeah we should. Just put this in the next PR.

Comment (rwongone) [reply handlers/handlers.go:60]: nvm I'll do it next PR

**Comment by rwongone** [comment]
LGTM again

**Pull request merged**